fe/fe-core/pom.xml

```diff
@@ -42,6 +42,12 @@ under the License.
         <jacoco.version>0.8.8</jacoco.version>
         <jprotobuf-precompile-plugin.version>2.2.12</jprotobuf-precompile-plugin.version>
         <python>python</python>
+        
+        <!-- async-profiler in UT -->
+        <async.profiler.enabled>${env.ASYNC_PROFILER_ENABLED:-true}</async.profiler.enabled>
+        <async.profiler.lib>${env.STARROCKS_HOME}/build-support/libasyncProfiler.so</async.profiler.lib>
```

Comment:
~~only enable it on x86 platform~~

Will this .so be available on arm platform as well?